docs/insomnia/get-started/install.md

```diff
@@ -19,65 +17,55 @@ Insomnia is available on macOS, Windows, and Linux. If you haven't already downl
     <div class="alert alert-primary">
     <b>Note</b>: The minimum macOS version supported is macOS 10.12 Sierra.
     </div>
-    Get Insomnia on macOS <a href="https://insomnia.rest/download">here</a>, or by using Brew.
-<br/><br/>
-<a href="https://insomnia.rest/download">Download</a> and double-click the disk image. When prompted, drag Insomnia to your <b>Applications</b> folder. This ensures future updates can be installed correctly.
-<br/><br/>
-macOS users can also install Insomnia using <a href="https://brew.sh/">Brew Cask</a> via the Insomnia package:
+    <a href="https://insomnia.rest/download">Download Insomnia on macOS</a> or use <a href="https://brew.sh/">Homebrew</a>:
```

Comment:
For reviewers: Is there an _easy_ way to reuse the ["Download Insomnia for macOS" from the website](https://insomnia.rest/download) here? I think the hyperlink is okay, just trying to think of visual designs that will make this content seem more professional?

Reply:
Hyperlink is ok